### DISCUSSION TITLE
Mobile TurboWarp

### DISCUSSION THREAD

**Discussion by Eaielectronic** [TurboWarp/extensions#1667] [Q&A]
Bonjour, j'ai amélioré un jeu dans TurboWrap pour mobile, mais il y a un problème : pour faire avancer le joueur, nous cliquons sur les boutons fléchés, mais lorsque nous appuyons sur deux boutons en même temps, un seul est activé. Je pense que c'est parce que TurboWarp considère le doigt comme la souris et que deux souris sont impossibles, mais sur mobile, nous devrions pouvoir cliquer sur deux boutons en même temps. Y a-t-il une extension pour résoudre ce problème ? Merci d'avance !!! Ligne de projet : https://eaielectronic.github.io/Electronix/Galery/Appel-The-Adventure/play-plan/

**Comment by Eaielectronic** [comment]
Hello, I improved a game in TurboWrap for mobile, but there is a problem: to move the player forward, we click on the arrow buttons, but when we press two buttons at the same time, only one is activated. I think it's because TurboWarp considers the finger as the mouse and two mice are impossible, but on mobile we should be able to click two buttons at the same time. Is there an extension to solve this problem? Thanks in advance!!! Project line: https://eaielectronic.github.io/Electronix/Galery/Appel-The-Adventure/play-plan/

**Reply by PURPL3-CAT** [reply]
I think Sensing+ is suited for this. It's designed for handling multiple mobile touches simultaneously.

**Reply by AmberEP** [reply]
With sensing+ i made tests for my project and when i put 2 fingers down on mobile it triggers the action i set to test that. On my touchscreen pc it also triggers. 
If there's "false" on your device it might be the problem, i think that like you said it's considered as a mouse pointer.

**Comment by Eaielectronic** [comment]
I looked, but the first block answers false for "supports sensing" on an 8th generation iPad. So I don't know...

**Reply by snowboyz0825** [reply]
just ignore the supports sensing because I've personally found it buggy and just take whether or not finger 1 is down, then you can make hitboxes that detects when I fingers x/y is within the arrow x/y

**Comment by AmberEP** [comment]
After checking, sensing+ extension is apparently just not supported on IOS yet. I'm not an expert so you should check the old discussions about that.

**Reply by GarboMuffin** [reply]
Sensing+ is supposed to work perfect on iOS since January

**Reply by AmberEP** [reply]
oh... it's weird then

**Comment by ampelectrecuted** [comment]
@GarboMuffin that iptv thing seems to be straight up piracy, and the wifi thing is likely for ip stealing, so delete them and lock the discussion
